### DISCUSSION TITLE
Raspberry Pi

### DISCUSSION THREAD

**Discussion by superlulu55** [mathoudebine/turing-smart-screen-python#93] [General]
Hi !

Thank you very much for this project. I bought this little cheap monitor screen, that works only for windows, and would monitoring my raspberry pi. This project helps me to do that !

I have installed Pynthon and all dependency and works great !
I made an ironman theme that i shared .

I think I have a little issue with Frequency et RAM available data, it seems wrong but i'm not sure.

![ironman](https://user-images.githubusercontent.com/79225820/203650255-48726735-0d84-4f4c-acf2-981db5f278c4.jpg)

**Comment by mathoudebine** [comment]
Hi @superlulu55!
You did a very nice job for your theme!
I will be happy to include it in this project if you want. For this, you can share the bitmaps and the theme.yaml file as a PR or directly in this chat.

For your metrics issues, what is your Raspberry Pi model?

**Reply by superlulu55** [reply]
Hi ! Thank you for your answer. I did it on the theme topic. There is the yaml file and png picture:
https://github.com/mathoudebine/turing-smart-screen-python/discussions/58#discussioncomment-4220246

I have a raspberry pi 3B model
